### ISSUE TITLE
你好，请问怎么评估都是0

### ISSUE THREAD

**Issue by zhw666888** [xiaolalala/Chinese-NER#4]

**Comment by xiaofei05** [comment]
我这边没有问题，可否详细描述一下

**Comment by zhw666888** [comment]
就是直接跑你的程序。结果p,r,f1基本全是0，而loss似乎又一直在算



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年6月25日(周五) 晚上7:27
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [xiaofei05/Chinese-NER] 你好，请问怎么评估都是0 (#4)




 
我这边没有问题，可否详细描述一下

**Comment by xiaofei05** [comment]
loss是正常在下降吗，main.py里面的labels列表和数据里的label对应吗，环境版本一致吗

**Comment by zhw666888** [comment]
loss是正常下降的，标签对应，其他没有报错，版本应该适配，我目前跑的bilstm CRF



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年6月25日(周五) 晚上7:53
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [xiaofei05/Chinese-NER] 你好，请问怎么评估都是0 (#4)




 
loss是正常在下降吗，main.py里面的labels列表和数据里的label对应吗，环境版本一致吗

**Comment by xiaofei05** [comment]
learning rate 使用的多少，是1e-2吗？

**Comment by zhw666888** [comment]
这些都是默认的



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年6月25日(周五) 晚上8:23
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [xiaofei05/Chinese-NER] 你好，请问怎么评估都是0 (#4)




 
learning rate 使用的多少，是1e-2吗？

**Comment by xiaofei05** [comment]
learning rate默认值是5e-5，是bert的默认值，运行Bilstm可以使用Readme中的命令，需要设置learning rate 为 1e-2。

**Comment by xiaofei05** [comment]
```py
# run bilstm
# it's better to reset the parameters
python main.py --model bilstm \
        --learning_rate 1e-2 \
        --num_train_epochs 20 \
        --train_batch_size 64 \
        --dev_batch_size 32

# run bilstm+crf
python main.py --model bilstm --crf \
        --learning_rate 1e-2 \
        --num_train_epochs 20 \
        --train_batch_size 64 \
        --dev_batch_size 32
```

**Comment by zhw666888** [comment]
现在可以了之前没注意这个学习率，之前按照bert的，学习率太小了，所以就感觉没用一样，感谢感谢



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年6月25日(周五) 晚上8:59
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [xiaofei05/Chinese-NER] 你好，请问怎么评估都是0 (#4)




 # run bilstm # it's better to reset the parameters python main.py --model bilstm \         --learning_rate 1e-2 \         --num_train_epochs 20 \         --train_batch_size 64 \         --dev_batch_size 32 # run bilstm+crf python main.py --model bilstm --crf \         --learning_rate 1e-2 \         --num_train_epochs 20 \         --train_batch_size 64 \         --dev_batch_size 32
